hexa.binding/processor/src/main/java/fr/lteconsulting/hexa/databinding/annotation/ObservableGwt.java

```diff
@@ -26,11 +27,23 @@
  * So if this annotation is put on a fr.lteconsulting.Person class,
  * it will generate an observable implementation with the following
  * fqn : fr.lteconsulting.ObservablePerson<br/><br/>
- * 
+ *
+ * @deprecated use {@link fr.lteconsulting.hexa.databinding.gwt.annotation.Observable}
```

Comment:
You can remove the deprecation comment here, but makes sense to use the same name now :+1: